tests/basic/sharedv4_svc_test.go

```diff
@@ -1880,3 +1880,145 @@ func runCmd(cmd string, n node.Node) (string, error) {
 	}
 	return output, err
 }
+
+var _ = Describe("{StickyVolumeTest}", func() {
+	var testrailID = 0
+	//testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/TBD
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("StickyVolumeTest", "Validate Sticky volume deletion failure", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var (
+		contexts []*scheduler.Context
+		volList  []*volume.Volume
+	)
+
+	Inst().AppList = []string{"nginx"}
+
+	stepLog := "should deploy & validate applications; update sticky flag and try deleting the volume"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("stickyvoltest-%d", i))...)
+		}
+
+		ValidateApplications(contexts)
+
+		Step("get volume list for all apps", func() {
+			log.InfoD("get volume list for all apps")
+			for _, ctx := range contexts {
+				vols, err := Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "Failed to list volumes for app: %s", ctx.App.Key)
+				volList = append(vols)
+			}
+		})
+
+		Step("update sticky flag for all volume(s) to 'on'", func() {
+			log.InfoD("update sticky flag for all volume(s) to 'on'")
+			for _, vol := range volList {
+				err = Inst().V.UpdateStickyFlag(vol.ID, "on")
+				log.FailOnError(err, "Failed to update sticky flag for volume: %s", vol.Name)
+			}
+		})
+
+		Step("get nodes & bounce volume driver", func() {
+			log.InfoD("get nodes & bounce volume driver")
+			for _, appNode := range node.GetStorageDriverNodes() {
+				stepLog = fmt.Sprintf("stop volume driver %s on node: %s",
+					Inst().V.String(), appNode.Name)
+				Step(stepLog,
+					func() {
+						log.InfoD(stepLog)
+						StopVolDriverAndWait([]node.Node{appNode})
+					})
+
+				stepLog = fmt.Sprintf("starting volume %s driver on node %s",
+					Inst().V.String(), appNode.Name)
+				Step(stepLog,
+					func() {
+						log.InfoD(stepLog)
+						StartVolDriverAndWait([]node.Node{appNode})
+					})
+
+				stepLog = "Giving few seconds for volume driver to stabilize"
+				Step(stepLog, func() {
+					log.InfoD(stepLog)
```

Comment:
I dont think this wait is required  as StartVolDriverAndWait will make sure PX is up before returning

Reply:
Removed this check